install/first_google_boot.sh

```diff
@@ -215,21 +219,42 @@ function extract_spinnaker_kube_credentials() {
     kube_zone=$MY_ZONE
   fi
 
+  if clear_metadata_to_file "kube_config" $config_path; then
```

Comment:
Take a look here :https://github.com/spinnaker/spinnaker/pull/1239/files/974284040f4e9941aeb38b4793b22956afa2a168#diff-6fe926a9170979345c6d40b1d595015aR253 
the enabled bit is only set after the call the gcloud produces the credentials file.